dev-client/src/screens/ScreenScaffold.tsx

```diff
@@ -48,26 +47,23 @@ export const ScreenScaffold = ({
   );
 
   return (
-    <BottomSheetModalProvider>
-      <SafeAreaView
-        style={[
-          styles.safeAreaContainer,
-          {backgroundColor: colors.primary.dark},
-        ]}>
-        <StatusBar
-          barStyle="light-content"
-          translucent
-          backgroundColor={'#00000000'}
-        />
-        <Column backgroundColor={colors.white} flex={1}>
-          <View onLayout={onLayout}>{PropsAppBar}</View>
-          <HeaderHeightContext.Provider
-            value={safeAreaTop + (headerHeight ?? 0)}>
-            <Box flex={1}>{children}</Box>
-          </HeaderHeightContext.Provider>
-        </Column>
-      </SafeAreaView>
-    </BottomSheetModalProvider>
+    <SafeAreaView
+      style={[
+        styles.safeAreaContainer,
+        {backgroundColor: colors.primary.dark},
+      ]}>
+      <StatusBar
+        barStyle="light-content"
+        translucent
+        backgroundColor={'#00000000'}
```

Comment:
question: can this be #000,  #000000 or a name color?

Reply:
#000 and #000000 mean different things (the extra 0s make it fully transparent) but it should be named yes, not actually part of the PR but i will update it